checkov/main.py

```diff
@@ -127,6 +129,8 @@ def run(banner=checkov_banner, argv=sys.argv[1:]):
             parser.error("The check ids specified for '--check' and '--skip-check' must be mutually exclusive.")
             return
 
+    integration_feature_registry.run_pre_scan()
```

Comment:
@mikeurbanski1 Is this protected if running without api-key?

Reply:
Each "integration feature" has logic to determine whether it runs, so nothing will happen if integration is not configured.